app/views/layouts/application.html.erb

```diff
@@ -35,6 +35,11 @@
   </head>
 
   <body class="govuk-template__body ">
+    <!-- Google Tag Manager (noscript) -->
+    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=<%= ENV['GOOGLE_TAG_MANAGER_TRACKING_ID'] %>"
+    height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+    <!-- End Google Tag Manager (noscript) -->
```

Comment:
Shouldn't this bit of code also be within a `if ENV['GOOGLE_TAG_MANAGER_TRACKING_ID'].present?` conditional?